cmd/kata-manager/kata-manager.sh

```diff
@@ -304,6 +304,14 @@ exec_document()
 
 	info "$msg"
 
+	# verify that we have OBS packages for the distro VERSION_ID
+	check_version=$(cat "${install_script}" | grep "${VERSION_ID}" || true)
```

Comment:
Nit: Again, you don't need this `cat`. You might want to anchor the match for a more reliable result too:

```sh
check_version=$(grep -E "\<${VERSION_ID}\>" "${install_script}" || true)
```

Reply:
@jodh-intel thanks , done :)